litmus/director/tcid-iuoi02-openebs-install/test.yml

```diff
@@ -95,34 +95,11 @@
         - name: Fetching project details
           set_fact:
             project_id: "{{ project_details.json.data[0].id }}"
-        
-        ## Fetch node details of the cluster
-        - name: Fetch node details of the cluster
-          uri:
-            url: "{{ director_url }}/v3/groups/{{ group_id }}/clusters/{{ cluster_id }}/nodes"
-            method: GET
-            url_username: "{{ username.stdout }}"
-            url_password: "{{ password.stdout }}"
-            force_basic_auth: yes
-            return_content: yes
-            body_format: json
-            status_code: 200
-          register: node_cluster
-
-        ## Fetch node-1 details of the cluster
-        - name: Fetch node-1 details of the cluster
-          set_fact:
-            node1_id: "{{ node_cluster.json.data[0].id}}"
-        
-        ## Fetch node-2 details of the cluster
-        - name: Fetch node-2 details of the cluster
-          set_fact:
-            node2_id: "{{ node_cluster.json.data[1].id}}"
 
         ## Labeling the node-1 of connected Cluster with controlPlaneNode=true and dataPlaneNode=true
         - name: Giving POST request on labelnodes
           uri:
-            url: "{{ director_url }}/v3/groups/{{ group_id }}/nodes/{{ node1_id }}/?action=labelnodes"
+            url: "{{ director_url }}/v3/groups/{{ group_id }}/nodes/{{ node_id[0] }}/?action=labelnodes"
```

Comment:
How are we getting the `node_id`?

Reply:
https://github.com/mayadata-io/oep-e2e/blob/43e765d375266ee96fdca2a8d110b328c7215cc4/litmus/director/tcid-iuoi02-openebs-install/test.yml#L71
From this block we are storing node_id